ui/app/src/state/actions/dialogs.ts

```diff
@@ -144,10 +144,11 @@ export const workflowCancellationDialogClosed = /*#__PURE__*/ createAction('WORK
 export const showRejectDialog = /*#__PURE__*/ createAction<Partial<RejectDialogStateProps>>('SHOW_REJECT_DIALOG');
 export const closeRejectDialog = /*#__PURE__*/ createAction<StandardAction>('CLOSE_REJECT_DIALOG');
 export const rejectDialogClosed = /*#__PURE__*/ createAction('REJECT_DIALOG_CLOSED');
+export const updateRejectDialog = /*#__PURE__*/ createAction<Partial<RejectDialogStateProps>>('UPDATE_REJECT_DIALOG');
 // endregion
 
 // region Legacy Form
-export const showEditDialog = /*#__PURE__*/ createAction<LegacyFormDialogStateProps>('SHOW_EDIT_DIALOG');
+export const showEditDialog = /*#__PURE__*/ createAction<Partial<LegacyFormDialogStateProps>>('SHOW_EDIT_DIALOG');
```

Comment:
Problem is partial makes all props optional and it isn't try that they all are. We want the typing system to yell if you try to dispatch without one of the required props. Why did you do this? If there's a prop that's optional, it should be declared as such directly on the props interface.

If needed, there can be an update action where everything is optional.

Reply:
let's discuss it tomorrow